### DISCUSSION TITLE
Unexpected exception occurred when trying to View Metrics on SSAS Model

### DISCUSSION THREAD

**Discussion by jbirrer06** [DaxStudio/DaxStudio#529] [Q&A] [answered]
First time posting, so hopefully I didn't miss this as a previous discussion...

I'm trying to view metrics on a new model that my team is building and it won't ever finish building the metrics/output.  It is a 2019 SSAS Model (about 18GB compressed in size).  We've checked the server and RAM is consistently at 77% (considering adding more), but never spikes while trying to analyze the model.  Any ideas why I'm unable to view the metrics?  Is there a compatibility issue with SSAS 2019 and DAX Studio?  I am able to analyze a model on SSAS 2017 (less than 3GB) with no issue, but that same model upgraded to 2019 fails. 

I get a more detailed error on the smaller model in 2019:

An unexpected error occurred (file 'xmtwdatacache.cpp', line 2723, function 'XMWindowingIterator::Init').

**Comment by marcosqlbi** [comment, marked as answer]
This is a bug in the Microsoft product, already reported by other users.
You should contact Microsoft to look for a fix. Every time you see "Unexpected error" you are facing a product bug.
We might find a workaround for DAX Studio and the View Metrics feature, but I'm not sure about the timing.

**Reply by jbirrer06** [reply]
Thanks for the quick response, @marcosqlbi.  Will follow up with MSFT.

**Comment by dgosbell** [comment]
For reference #459 is the issue we are tracking where view metrics seems to hit a bug in the tabular engine for some models
